C++20: cannot convert from 'initializer list' to 'xbox::httpclient::XPlatSecurityInformation'

https://github.com/microsoft/libHttpClient/blob/cafedfde1da13a64b865a100adef0868cf7f8c88/Source/HTTP/WinHttp/winhttp_provider.cpp#L325

error C2440: '<function-style-cast>': cannot convert from 'initializer list' to 'xbox::httpclient::XPlatSecurityInformation' 

https://stackoverflow.com/questions/57271400/why-does-aggregate-initialization-not-work-anymore-since-c20-if-a-constructor

--- a/Source/HTTP/WinHttp/winhttp_provider.h
+++ b/Source/HTTP/WinHttp/winhttp_provider.h
@@ -46,6 +46,10 @@ struct XPlatSecurityInformation
     XNetworkingSecurityInformation* securityInformation{ nullptr };
 #endif
     uint32_t enabledHttpSecurityProtocolFlags;
+    XPlatSecurityInformation(uint32_t flags)
+    {
+        enabledHttpSecurityProtocolFlags = flags;
+    }
 };
 
 class WinHttpProvider

--- a/Source/HTTP/WinHttp/winhttp_provider.h
+++ b/Source/HTTP/WinHttp/winhttp_provider.h
@@ -45,7 +45,7 @@ struct XPlatSecurityInformation
     http_internal_vector<uint8_t> buffer;
     XNetworkingSecurityInformation* securityInformation{ nullptr };
 #endif
-    uint32_t enabledHttpSecurityProtocolFlags;
+    uint32_t enabledHttpSecurityProtocolFlags { 0 };
     XPlatSecurityInformation(uint32_t flags)
     {
         enabledHttpSecurityProtocolFlags = flags;

